modules/tasks/tasks/src/main/java/org/motechproject/tasks/service/impl/TaskInitializer.java

```diff
@@ -82,6 +84,20 @@ public boolean evalConfigSteps(Map<String, DataProvider> dataProviders) throws T
         return result;
     }
 
+    public int getActionFilters() {
```

Comment:
this method looks complicated, try to simplify it. It is more readable to use only one return statement